crates/release_plz_core/src/command/release.rs

```diff
@@ -754,8 +755,19 @@ fn last_changelog_entry(req: &ReleaseRequest, package: &Package) -> String {
 
 #[cfg(test)]
 mod tests {
+    use std::sync::Mutex;
+
+    use lazy_static::lazy_static;
+
+    use fake_package::metadata::fake_metadata;
+
     use super::*;
 
+    lazy_static! {
+        // Trick to avoid the tests to run concurrently
+        static ref NO_PARALLEL: Mutex<()> = Mutex::default();
+    }
```

Comment:
you don't want tests to run concurrently because you don't want the env var to be set in other tests?
If yes, can you add a comment to explain this?

Reply:
Yes, exactly. I'll update the comment 👍